compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/reanimated-shared-value-writes.expect.md

```diff
@@ -36,21 +36,22 @@ import { useSharedValue } from "react-native-reanimated";
  * of render
  */
 function SomeComponent() {
-  const $ = _c(3);
+  const $ = _c(2);
   const sharedVal = useSharedValue(0);
-
-  const T0 = Button;
-  const t0 = () => (sharedVal.value = Math.random());
-  let t1;
-  if ($[0] !== T0 || $[1] !== t0) {
-    t1 = <T0 onPress={t0} title="Randomize" />;
-    $[0] = T0;
+  let t0;
+  if ($[0] !== sharedVal) {
+    t0 = (
+      <Button
+        onPress={() => (sharedVal.value = Math.random())}
```

Comment:
seems like the check wasn't completely redundant since we are getting different output. why does this output change exactly?

Reply:
Ahhh makes sense. Cool